Tensile/Tests/extended/convolution_config/YamlBuilder/YamlBuilder.py

```diff
@@ -27,6 +27,8 @@
 import copy, operator, pytest
 from functools import reduce
 from Tensile.SolutionStructs import ConvProblem
+from Tensile.Utilities.ConditionalImports import yamlDumper
+
 import yaml
```

Comment:
NBD but seems like after your change the yaml is not needed and can be removed from imports

Reply:
It is needed to qualify calls like `dump` and `load` no?

Reply:
yes, you are correct I missed that